update_compilers/install_binaries.sh

```diff
@@ -89,6 +90,54 @@ if [[ ! -d ${OPT}/x86-to-6502/lefticus ]]; then
     rm -rf /tmp/build
 fi
 
+#########################
```

Comment:
(this is fine!) but we should probably think about moving some of this stuff to misc-builder. I can take a look at that, I'd like to make it easier to do that instead of this stuff :)

Reply:
A builder would indeed be nice, but it is quite involved in its current state. Not sure what a more friendly misc-builder would look like

Reply:
although... maybe we can integrate a simpler version of the library building process; a yaml with the name, url, source, destination, version etc, and then a script just make/cmake's it if its not on S3 yet or something?